lighthouse-core/audits/seo/link-text.js

```diff
@@ -30,7 +30,7 @@ class LinkText extends Audit {
       description: 'Links have descriptive text.',
       failureDescription: 'Links do not have descriptive text',
       helpText: 'Descriptive link text helps search engines understand your content. ' +
-      '[Learn more](https://webmasters.googleblog.com/2008/10/importance-of-link-architecture.html)',
+      '[Learn more](https://webmasters.googleblog.com/2008/10/importance-of-link-architecture.html).',
```

Comment:
unrelated change, sorry to shove it here, but it was killing me

<img width="712" alt="had_to_fix_it" src="https://user-images.githubusercontent.com/985504/34506104-238f80d2-f02a-11e7-87f0-3cf8592680f9.png">

Reply:
😄 👍